webapp/packages/core-utils/src/toSafeHtmlString.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+export function toSafeHtmlString(dirty: string): string {
```

Comment:
Probably better to use lib like dompurify to sanitize html

Reply:
this was my first solution. @Wroud suggested to generate a html safe string
in this case we kill 2 rabbits:
- the code is not executable by browser
- the user will see that something illegal happened in his JSON tab editor and start investigation what happened
 
![image](https://github.com/dbeaver/cloudbeaver/assets/26230285/72c9d8f2-7044-4778-a8ef-7653d9ec7c73)
![image](https://github.com/dbeaver/cloudbeaver/assets/26230285/5c202c32-780d-44c1-bb77-091d42fd7592)